CHANGELOG.md

```diff
@@ -264,6 +264,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `BaseFinetuning` callback to properly handle parent modules w/ parameters ([#7931](https://github.com/PyTorchLightning/pytorch-lightning/pull/7931))
 
 
+- Fixed `PyTorchProfiler` generated filenames ([#8009](https://github.com/PyTorchLightning/pytorch-lightning/pull/8009))
```

Comment:
Can you elaborate on what's the fix exactly? Or what was broken

Reply:
Hey @carmocca,
The filenames looked like fit-None.{some_number} and the action name for PyTorch Profiler wasn't saved, creating overlap in TensorBoard Plugin.